actors/runtime/src/test_utils.rs

```diff
@@ -245,6 +245,31 @@ pub struct ExpectComputeUnsealedSectorCid {
     exit_code: ExitCode,
 }
 
+pub fn expect_abort_contains_message<F: FnOnce() -> Result<RawBytes, ActorError>>(
```

Comment:
I went with trait FnOnce since I knew how to create closures with the behavior that I wanted and this seems like a minimal interface since expect abort calls will only call the input function once.  Please advise if there's something more idiomatic or better.

Reply:
Well, given that the function is called up-front and takes no arguments, I'd expect to just pass in the result. I.e.:

```rust
pub fn expect_abort_contains_message(exit_code: ExitCode, expect_msg: &str, result: Result<RawBytes, ActorError>>) { ... }
```

Reply:
I agree with steven here, no need for the closure. Lets just pass the result.